src/evaluator.ts

```diff
@@ -92,6 +105,72 @@ export class Evaluator {
     }
   }
 
+  private isTargetInList(target: Target, targets: Target[]): boolean {
+    return (
+      targets.find((elem) => elem.identifier === target.identifier) != undefined
+    );
+  }
+
+  private convertApiTargetsToEvalTargets(targets: ApiTarget[]): Target[] {
+    return targets.map(
+      (elem) =>
+        ({
+          identifier: elem.identifier,
+          name: elem.name,
+          anonymous: elem.anonymous,
+          attributes: elem.attributes,
+        } as Target),
```

Comment:
Do you need this type assertion? The return type on the method should be sufficient

Reply:
compiler complains about mismatched types